microsoft/atlas-design
# Add form behavior

Task: task-[601375](https://dev.azure.com/ceapex/Engineering/_workitems/edit/625202/?workitem=601375)

Link: preview-[408](https://design.docs.microsoft.com/pulls/408/components/form.html)

This PR adds the form behavior component to Atlas to handle form validation.

## Testing

1. Visit https://design.docs.microsoft.com/pulls/408/components/form.html and review the form behavior section.
2. Check code changes.
